experimental/traffic-portal/src/app/core/cache-groups/phys-loc/detail/phys-loc-detail.component.html

```diff
@@ -0,0 +1,78 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card>
+	<tp-loading *ngIf="!physLocation"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="physLocation">
+		<mat-card-content>
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required [(ngModel)]="physLocation.name" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Short Name</mat-label>
+				<input matInput type="text" name="shortName" required [(ngModel)]="physLocation.shortName" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Address</mat-label>
+				<input matInput type="text" name="address" required [(ngModel)]="physLocation.address" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>City</mat-label>
+				<input matInput type="text" name="city" required [(ngModel)]="physLocation.city" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>State</mat-label>
+				<input matInput type="text" name="state" required [(ngModel)]="physLocation.state" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Zip</mat-label>
+				<input matInput type="text" name="zip" required [(ngModel)]="physLocation.zip" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Poc</mat-label>
```

Comment:
Poc stands for "Point of Contact", which should be spelled out because otherwise nobody knows what that means.